### DISCUSSION TITLE
i want to remove fr language

### DISCUSSION THREAD

**Discussion by Samrridh** [mearashadowfax/ScrewFast#247] [Q&A] [answered]
i want to keep my site simple and only use english. How may i do so

i have delelted the 'fr' folder, but i also want to delete the fr navigation.ts and it is giving errors as it is also used in footer and other components

**Comment by mearashadowfax** [comment, marked as answer]
Hey @Samrridh! Just change the branch to a monolingual one.

**Reply by Samrridh** [reply]
by removing this from astro config or by some other method?
![image](https://github.com/user-attachments/assets/e5cb8595-3d60-4e32-a9db-e931f5be7200)

**Reply by mearashadowfax** [reply]
> by removing this from astro config or by some other method?

You can just use [monolingual branch](https://github.com/mearashadowfax/ScrewFast/tree/monolingual-site), which is set up for a single language. If you're not using Starlight Docs, you can remove all docs-related configurations, including the code snippet above.
